### PR TITLE
#63 - Rebuild UI for Contact Form.

### PR DESCRIPTION
**Pull request by deep-a**
Task completed, please review and suggest changes. Thanks.

### CODE REVIEW

File: src/pages/contact.jsx [hunk 1]
```diff
@@ -1,21 +1,99 @@
...
-      </div>
+    <BaseLayout pageTitle="Contact Donate Gifts">
+      <section id="contact" className="mb-5">
+        <div className="d-flex flex-wrap pt-5 pb-4">
+          <div className="row d-flex">
+            <div className="col-12 text-center">
+              <h2 className="">Get in Touch</h2>
+              <p>Please fill out form and we will be in touch as soon as possible.
+                Perhaps your question has been answered, see our <a
+                href="/faq">FAQ</a> page.</p>
```
Comment (Enubia) [inline src/pages/contact.jsx:35]: use the Link component from next here

**Review by Enubia — CHANGES_REQUESTED** [review]
looks good overall

just two things
- make use of the translations
- merge the master after #88 was merged as this one includes some stuff that you probably need

File: src/pages/contact.jsx (hunk 1 above)
Comment (deep-a) [reply src/pages/contact.jsx:35]: Can you please explain what you mean by using translations more? I switched to using the Link component and merged #88.

Comment (Enubia) [reply src/pages/contact.jsx]: there are translation files located under ```/public/locales/en/```
bunch of json files, one for each page we have

please use them, example can be taken from the ```howTo.json``` and the howto.jsx

Comment (deep-a) [reply src/pages/contact.jsx]: Thanks for the info, please review the changes.

**Review by Enubia — APPROVED** [review]

**Pull request merged**